docs/advanced/firewall.md

```diff
@@ -21,6 +21,27 @@ Snikket currently requires the following ports to be open/forwarded:
  | 5349/5350     | Audio/Video Data Proxy Negotiations and IP Discovery over TLS <br /> (STUN/TURN over TLS) |
 
 
- |**UDP only**   |                                                                                    |
- | :------------ | :----------------------------------------------------------------------------------|
- | 49152-65535   | Audio/Video Data Proxy (Turn Data)                                                 |
+ |**UDP only**  |                                                                                    |
+ | :----------- | :----------------------------------------------------------------------------------|
+ | 49152-65535  | Audio/Video Data Proxy (Turn Data, see below)                                      |
+
+
+## Changing the turnserver port range
+
+The STUN/TURN server is required for audio/video (A/V) calls to work reliably on all kinds of "difficult" client networks. For this, a relay connection is established which routes the (encrypted) A/V data via your Snikket server. As generally the number of concurrent calls is not known and it needs to compete with ports already in use on the machine, the TURN server defaults to a range with a high number of ports (about 16 thousand).
```

Comment:
It's implied, but not explicitly stated, that more ports == more concurrent calls. I think it would be good to clarify that a smaller range means fewer concurrent calls (do we know roughly how many?). It's often asked why so many ports are needed.

Maybe worth an FAQ entry as well actually...

Reply:
Thanks, I added some text. Might be too much text, or too technical, or not technical enough. Interested in more feedback.

Reply:
I think it's about right. Possibly a bit wordy for this document, but what I will probably do is borrow some of the text for the FAQ entry, and link to that from here or something (or maybe the other way around... hmm). LGTM for now though.